### PR TITLE
f2fs, squashfs, nfs4

### PR DESCRIPTION
**Pull request by ojaksch**
f2fs, squashfs and nfs4 enabled. No yet tested on MiSTer, but I can't see any hindrance why it shouldn't work.
f2fs settings taken from https://wiki.gentoo.org/wiki/F2FS
I have to update my ARM toolchain, but this config is a good starting point.
EDIT: Built kernel transferred to one of my MiSTer, is booting and working fine and I can use my NFS4 mount. Hooray!

### CODE REVIEW

**Comment by amstan** [comment]
Cool. Good job!

Yeah, I missed nfs4 in my previous commit.

Some suggestions for your change (though I know you probably didn't get to doing the whole thing):
* You need to increment [pkgrel](https://github.com/amstan/MiSTerArch/blob/main/linux-mister/PKGBUILD#L11) every time you make such a substantial change (otherwise none of the pacmans on people's setups will know there's any update, since the version will stay the same)
* [sha256sum](https://github.com/amstan/MiSTerArch/blob/main/linux-mister/PKGBUILD#L24) needs to be refreshed for the new config file now
* Commit message should probably include a few words. First 2 lines of your PR message might be a good starting point.
    * I suggest also adding a BUG=Issue #8 line as well to link to the bug.
    * Maybe add a TEST= line for how you did the NFS booting, is it just editing uboot.txt for the new rootfs? Or are you doing this for other storage like /media/fat, so it's doable with /etc/fstab?

See https://github.com/amstan/MiSTerArch/commit/e4b805a360d83669454839261410b87188c6535a as a template for a similar change.

Anyway, even if you don't get to it, this helps me a lot (since I don't have to wonder which configs I need, and apparently you just tested it too so that's even better!) and I could take it over if you wish.

**Comment by amstan** [comment]
Wait, I just realized. You're fixing @solskogen's bug, that's not even your bug. So that's even more appreciated!

**Comment by ojaksch** [comment]
You're welcome! I'll have a look to your suggestions after breakfast (it's Sat. 8:30am here :) )
While being here for NFS4 i just solved @solskogen's issues as well and also included squashfs. I have no need for the latter, but it might be useful in future...

**Comment by ojaksch** [comment]
> Some suggestions for your change (though I know you probably didn't get to doing the whole thing):
[...] pkgrel, sha256sum

Gosh, you are so right. My changes are belonging to an Arch's PKGBUILD and not "kernel only". Embarrassing for someone who has been building packages for so long.

File: linux-mister/config [hunk 1]
```diff
@@ -3985,7 +4017,6 @@ CONFIG_DEBUG_BUGVERBOSE=y
 #
 CONFIG_DEBUG_INFO=y
 # CONFIG_DEBUG_INFO_REDUCED is not set
-# CONFIG_DEBUG_INFO_COMPRESSED is not set
```
Comment (amstan) [inline linux-mister/config:3988]: Try to not touch lines like these. It caused my PKGBUILD to start asking questions about DEBUG_INFO and KASAN.

**Pull request closed without merging**